installers/beamsim-jupyter/radiasoft-download.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
```

Comment:
The installer name is confusing because it neither installs beamsim codes nor jupyter. This is a specialized installer, which could be documented as "pip-install-ml". We have installers like "sirepo", "jupyter", etc. that install the docker images and run them. One might think that this installer as named would download and run the beamsim-jupyter docker image.

Reply:
I don't like `pip-install-ml`. I think both `pip` and `ml` set unnecessary and incorrect (ex seaborn is not strictly for ml) expectations. How about `jupyter-scientific-packages`?

Reply:
- These have nothing to do with Jupyter. They can be used outside Jupyter just fine.
- The install is specifically for python related packages, and they do use pip (vs conda) so I think pip describes what this is doing specifically, and differentiates from other types of installers.
- Everything we do with beamsim is scientific :)
- Packages is superfluous

How about `py-ml-and-coupled`? The problem is that there's a coupling between the ml packages and seaborn, etc.

Reply:
> The install is specifically for python related packages, and they do use pip (vs conda) so I think pip describes what this is doing specifically, and differentiates from other types of installers.

I guess that is a distinction I didn't know we were enforcing. I'm not sure why it is useful to say "we can only install python here".

> Everything we do with beamsim is scientific :)

Yes, not a great name. My hope was to distinguish between deps like jupyterlab which are more for the "service" and less for the "science".

> Packages is superfluous

Agreed.

> How about py-ml-and-coupled? The problem is that there's a coupling between the ml packages and seaborn, etc.

I'm hung up on ml being in the name. I don't understand why the packages in this installer __have__ to have something to do with ml?

Reply:
I think it is replacing ml.sh almost entirely, and frankly, it seems like it should replace it all.

>    install_pip_install h5ImageGenerator \
>        scikit-learn sympy \
>        tensorflow~=2.15.0

I think this is confusing in ml.sh. I think we should build two rpms rscode-ml-cpu and rscode-ml-gpu. perhaps they could provide "rscode-ml" so the dependencies make sense, and gpu-jupyter docker image should replace rscode-ml-cpu with rscode-ml-gpu.

I think we should installing over an rscode rpm. Indeed, it might make sense for beamsim-jupyter to be built with other rpms, and not have installs of its own except for configuration files that vary from sirepo. Even those could be rpms.

If you need to release this, do that. However, I think this has been a mess for a while, and we need to figure out how to make it clear/clean.

Reply:
After discussion: We are going to move all ml deps into this installer. Then ml.sh will call this installer (gpu vs non-gpu). Packages like seaborn will go back to container-beamsim-jupyter